pkg/catalog/catalog.go

```diff
@@ -9,55 +9,63 @@ import (
 )
 
 // List lists all the available component types
-func List(client *occlient.Client) ([]string, error) {
+// returned map has key: imageStream and serviceCatalog and value is slice of respective components
+func List(client *occlient.Client) (map[string][]string, error) {
 	var catalogList []string
 	imageStreams, err := getDefaultBuilderImages(client)
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to get image streams")
 	}
 	catalogList = append(catalogList, imageStreams...)
+	finalList := make(map[string][]string)
+	finalList["imageStream"] = catalogList
+	clusterServiceClasses, err := client.GetClusterServiceClassExternalNames()
+	if err != nil {
+		log.Debug("Unable to get cluster serviceClassExternalName")
+	}
+	finalList["serviceCatalog"] = clusterServiceClasses
 
-	// TODO: uncomment when component create supports template creation
-	//clusterServiceClasses, err := client.GetClusterServiceClassExternalNames()
-	//if err != nil {
-	//	return nil, errors.Wrap(err, "unable to get cluster service classes")
-	//}
-	//catalogList = append(catalogList, clusterServiceClasses...)
-
-	return catalogList, nil
+	return finalList, nil
 }
 
 // Search searches for the component
 func Search(client *occlient.Client, name string) ([]string, error) {
 	var result []string
-	componentList, err := List(client)
+	catalogList, err := List(client)
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to list components")
 	}
 
 	// do a partial search in all the components
-	for _, component := range componentList {
-		if strings.Contains(component, name) {
-			result = append(result, component)
+	for _, catalog := range catalogList {
+		for _, component := range catalog {
+			if strings.Contains(component, name) {
+				result = append(result, component)
+			}
 		}
+
 	}
 
 	return result, nil
 }
 
 // Exists returns true if the given component type is valid, false if not
-func Exists(client *occlient.Client, componentType string) (bool, error) {
+// Exists returns catalog type either `imageStream` or `serviceCatalog`
```

Comment:
Please update this comment, it says "Exits returns" twice. 
The message should clearly describe what this function does and that each return value means